gapir/cc/context.cpp

```diff
@@ -231,13 +231,14 @@ void Context::registerCallbacks(Interpreter* interpreter) {
           // drop them and try to create VkInstance again.
           if (Vulkan::hasValidationLayers(pCreateInfo->ppEnabledLayerNames,
                                           pCreateInfo->enabledLayerCount) ||
-              Vulkan::hasDebugReportExtension(
+              Vulkan::hasDebugReportOrDebugUtilsExtension(
```

Comment:
The strategy to keep adding to this function name is not going to work out. Let's make it more general and call it `hasDebugExtension`.

Reply:
Agreed! I hesitated, since debug report and debug utils seems to be the only two debug-related extensions existing so far, so I though it might be OK to stay explicit at the price of this mouthful. But `hasDebugExtension` is also fine with me.